src/Module.jl

```diff
@@ -26,6 +26,7 @@ function check_parent(M::FPModule{T}, N::FPModule{T}) where T <: RingElement
 end
 
 is_finite(M::FPModule{<:FinFieldElem}) = true
+is_finitely_generated(M::FPModule) = true
 
```

Comment:
```suggestion
is_finitely_generated(M::Module) = isfinite(ngens(M)) || throw(NotImplementedError(:is_finitely_generated, M))

```
we could add this line to make the change nonbreaking for downstream